styles/subpage-style.css

```diff
@@ -0,0 +1,175 @@
+/* GENERAL */
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  --background-color: #E5EAFA;
+  --text-color: #25554A;
+  --font-family: 'Courier New', Courier, monospace;
+  --font-size: 1vw;
+}
+
+body {
+  background-color: var(--background-color);
+  color: var(--text-color);
+  min-height: 100vh;
+  display: flex;
+  flex-direction: column;
+}
+
+a {
+  text-decoration: none;
+  color: inherit;
+}
+
+a:hover{
+  color: red;
+}
+
+h1 {
+  font-family: 'Times New Roman', Times, serif;
+  text-align: center;
+  font-size: 3em;
+  border: 3px solid blue;
+  padding: 5px 10px;
+  width: fit-content;
+  margin: 30px auto 50px auto;
+}
+
+p {
+  font-family: 'Times New Roman', Times, serif;
+  color: rgb(44, 46, 45);
+  line-height: 1.4em;
+}
+
+ul {
+  list-style: none;
+  padding-top: 10px;
+  padding-bottom: 10px;
+}
+
+main {
+  max-width: 1000px;
+  margin: 0 auto;
+  padding: 10px;
+}
+
+.separator {
+  width: 90vw;
+  align-self: center;
+  border-bottom: solid 2px;
+  margin: 30px 0px;
+}
+
+footer {
+  font-size: .7em;
+  font-family: 'Courier New', Courier, monospace;
+  text-align: center;
+}
+
+/* NAV */
+
+nav {
+  font-family: 'Courier New', Courier, monospace;
+  width: 90vw;
+  height: 50px;
+  margin: 0 auto;
+  background-color: yellow;
+}
+
+#nav-list {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  flex-wrap: wrap;
+  margin-top: 2px;
+}
+
+.nav-item {
+  display: inline-block;
+  flex: auto;
+  font-size: 1em;
+  text-align: center;
+  padding-top: 10px;
+  padding-bottom: 10px;
+}
+
+
+/* ABOUT */
+
+#about {
+  height: 70vh;
+  padding: 1rem 8rem;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  align-items: center;
+}
+
+#about img {
+  width: 26%;
+  height: auto;
+}
+
+#about h2 {
+  width: 22%;
+  word-wrap: normal;
+  font-size: 2.2vw;
+  color: black;
+  font-weight: 200;
+}
+
+#about ul {
+  width: 30%;
+  padding: 0;
+  list-style: none;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+#about li {
+  font-family: 'Courier New', Courier, monospace;
+  font-size: 1vw;
+  background-color: blue;
+  color: white;
+  height: fit-content;
+  width: fit-content;
+  padding: 15px;
+  border-radius: 20px;
+  margin-bottom: 10px;
+  margin-right: 10px;
+  text-align: center;
+}
+
+
+/* PORTFOLIO */
+
+#portfolio {
+  font-family: 'Courier New', Courier, monospace;
+  display: flex;
+  flex-flow: row wrap;
+  height: 60vh;
```

Comment:
This line is causing a small layout bug - if the browser is dragged narrower, the flex content will spill over the footer.  Removing this allows the height of the main content to expand and pushes the footer down to the bottom of the page.
```suggestion
  
```